huaweicloud/services/cts/resource_huaweicloud_cts_tracker.go

```diff
@@ -137,6 +137,14 @@ func resourceCTSTrackerCreate(ctx context.Context, d *schema.ResourceData, meta
 
 	d.SetId(resourceID)
 
+	if rawTag := d.Get("tags").(map[string]interface{}); len(rawTag) > 0 {
+		tagList := expandResourceTags(rawTag)
```

Comment:
ditto

Reply:
Cannot use this method because the types of tags are different